platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -522,17 +356,13 @@ public class SwirldsPlatform implements Platform, ConnectionTracker, Startable {
                 actualMainClassName,
                 selfId,
                 swirldName,
-                this::createPrioritySystemTransaction,
+                txn -> this.createSystemTransaction(txn, true),
                 this::haltRequested,
                 appCommunicationComponent,
                 preConsensusEventWriter,
                 currentPlatformStatus::get);
         wiring.registerComponents(components);
 
-        final NetworkStatsTransmitter networkStatsTransmitter =
```

Comment:
The directive was to keep `NetworkStatsTransmitter` - did that not include continuing to actually transmit these?

Reply:
This transmitter was adding a nasty circular dependency. I didn't delete the code, but there is no need to actually transmit these transactions until we take the time to write handlers.